README.md

```diff
@@ -11,3 +11,52 @@ Make sure that you have the compatible versions of [Visual Studio 2019 (version
 ## How to run this application
 To run this application, you need to first clone the `getting-started-with-the-blazor-progressbar` repository and then open it in Visual Studio 2019. Now, simply build and run your project to view the output.
 
+## Features and Benefits
+
+## Indeterminate
+Use the indeterminate mode progress bar when progress cannot be estimated or calculated. It can be combined with determinate mode to let users know that the app is estimating the progress before the actual progress starts.
+
+## Segments
+Avoid writing several lines of code to create and position multiple progress bars: just divide a progress bar into multiple [segments](https://blazor.syncfusion.com/documentation/progress-bar/customization#segments) using a single API to visualize the progress of multiple sequential tasks.
+
+## Visualize multiple ranges
+Along with visualizing the progress of a task, users can also visualize multiple ranges with gradient colors or solid colors that are mapped to each range to enhance the readability.
+
+## Related links
+
+[Learn More about Blazor ProgressBar](https://www.syncfusion.com/blazor-components/blazor-progressbar?utm_source=github&utm_medium=listing&utm_campaign=blazor-progressbar-github-samples)
+
+[Download Free Trial](https://www.syncfusion.com/downloads/blazor?utm_source=github&utm_medium=listing&utm_campaign=blazor-progressbar-github-samples)
+
+[Pricing](https://www.syncfusion.com/sales/products/blazor?utm_source=github&utm_medium=listing&utm_campaign=blazor-progressbar-github-samples)
+
+[Documentation](https://blazor.syncfusion.com/documentation/progressbar/getting-started?utm_source=github&utm_medium=listing&utm_campaign=blazor-progressbar-github-samples)
+
+[View Demos](https://github.com/SyncfusionExamples/getting-started-with-the-blazor-progressbar?utm_source=github&utm_medium=listing&utm_campaign=blazor-progressbar-github-samples)
+
+[Community Forums](https://www.syncfusion.com/forums/blazor-components?utm_source=github&utm_medium=listing&utm_campaign=blazor-progressbar-github-samples)
+
+[Suggest a feature or report a bug](https://www.syncfusion.com/feedback/blazor-components?utm_source=github&utm_medium=listing&utm_campaign=blazor-progressbar-github-samples)
+
+[Online example](https://blazor.syncfusion.com/demos/toolbar/default-functionalities?utm_source=github&utm_medium=listing&utm_campaign=blazor-progressbar-github-samples)
+
+[Video Tutorials](https://www.syncfusion.com/tutorial-videos/blazor/toolbar?utm_source=github&utm_medium=listing&utm_campaign=blazor-progressbar-github-samples)

```

Comment:
this is progressbar control right?
![image](https://user-images.githubusercontent.com/61148953/175477090-a6b43851-b1d1-4391-ad2e-f5887842a84e.png)

Reply:
control name changed